test/validation-ui-tests.js

```diff
@@ -625,6 +625,74 @@ QUnit.test('Issue #44 - Validation Element - Is Invalid Test', function(assert)
 
 });
 
+QUnit.test('Validatable can become non-validatable', 0, function(assert) {
```

Comment:
Instead of using this syntax (0 expectations), why not add an assert at the end of the test, like `assert.equal(ko.validation.utils.isValidatable(false), false);` IMO, It would better describe the test.